lib/pages/perkPage.dart

```diff
@@ -168,11 +204,13 @@ class _PerkPageState extends State<PerkPage> {
                       Transform.scale(
                         scale: 1.5,
                         child: Switch(
-                          value: killerMode,
+                          value: perkModeSwitch,
                           onChanged: (value) async {
                             setState(() {
-                              killerMode = value;
-                              setPerkListAndRandomise();
+                              perkModeSwitch = value;
+
```

Comment:
Blank line here that can be removed from what I can see?

Reply:
There's only 1 blank line.